filebeat/docs/autodiscover-hints.asciidoc

```diff
@@ -148,3 +148,6 @@ You can label Docker containers with useful info to spin up {beatname_uc} inputs
 
 The above labels configure {beatname_uc} to use the Nginx module to harvest logs for this container.
 Access logs will be retrieved from stdout stream, and error logs from stderr.
+
+To ensure compliance with docker label key recommendations the separator between the prefix and 
+hints config key is configurable via the key `separator`.
```

Comment:
I think it'd worth to add a reference to Amazon ECS, as this is probably the main use case for this feature, it'd be also good to add an example, and please mention also the valid separators.